### PR TITLE
2024-01-16-explain_clinical_doc_oncology_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

File: docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_generic_en.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+| 21 | a bone marrow aspiration                 | PROBLEM    | Hypothetical |
+| 22 | the procedure                            | TREATMENT  | Hypothetical |
+| 23 | usefulness of the bone marrow            | TEST       | Hypothetical |
+| 24 | refractory anemia                        | PROBLEM    | Present      |
+| 25 | therapeutic interventions                | TREATMENT  | Hypothetical |
+| 26 | the studies                              | TEST       | Planned      |
+| 27 | an ultrasound of his abdomen             | TEST       | Planned      |
+| 28 | fullness of the spleen                   | PROBLEM    | Possible     |
+
+# Relation Extraction Result
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_generic_en.md:130]: filter O labels in RE results.

File: docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_generic_en.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+|  1 | mild arcus senilis in the right          | PROBLEM    | Present      |
+|  2 | jugular venous pressure distention       | PROBLEM    | Absent       |
+|  3 | adenopathy                               | PROBLEM    | Absent       |
+|  4 | tender                                   | PROBLEM    | Absent       |
+|  5 | some fullness in the left upper quadrant | PROBLEM    | Possible     |
+|  6 | some edema                               | PROBLEM    | Present      |
+|  7 | clubbing                                 | PROBLEM    | Absent       |
+|  8 | refractory anemia                        | PROBLEM    | Present      |
+|  9 | transfusion                              | TREATMENT  | Present      |
+| 10 | B12                                      | TREATMENT  | Planned      |
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_generic_en.md:110]: Check this sentence again, these should be PRESENT

File: docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_generic_en.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+| 11 | iron                                     | TREATMENT  | Planned      |
+| 12 | folic acid                               | TREATMENT  | Possible     |
+| 13 | Procrit                                  | TREATMENT  | Planned      |
+| 14 | blood loss                               | PROBLEM    | Absent       |
+| 15 | the previous esophagogastroduodenoscopy  | TEST       | Past         |
+| 16 | His creatinine                           | TEST       | Past         |
+| 17 | an underlying myelodysplastic syndrome   | PROBLEM    | Possible     |
+| 18 | bone marrow failure                      | PROBLEM    | Possible     |
+| 19 | His creatinine                           | TEST       | Absent       |
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_generic_en.md:119]: incorrect

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

File: docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_vop_en.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+He put me on two medications - I take metformin 500 mg twice a day, and glipizide 5 mg before breakfast and dinner. I also have to watch what I eat and try to exercise more.
+Now, I also have chronic acid reflux disease or GERD. Now I take a daily pill called omeprazole 20 mg to reduce the stomach acid and control the heartburn symptoms.""")
+
+```
+</div>
+
+## Results
+
+```bash
+#NER and Assertion Status Results
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_vop_en.md:74]: shared a request via DM

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

File: docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_oncology_en.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+edition: Healthcare NLP 5.2.1
+spark_version: 3.0
+supported: true
+annotator: PipelineModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
```
[the remark this answers is not in the document]
Comment (ahmet-mesut) [reply docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_oncology_en.md:19]: Updated according to DM

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

File: docs/_posts/bunyamin-polat/2024-01-17-atc_resolver_pipeline_en.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
+---
+layout: model
+title: Pipeline for ATC Sentence Entity Resolver
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-17-atc_resolver_pipeline_en.md:3]: @akrztrk update here

File: docs/_posts/bunyamin-polat/2024-01-17-cpt_procedures_measurements_resolver_pipeline_en.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
+---
+layout: model
+title: Pipeline for CPT Sentence Entity Resolver
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-17-cpt_procedures_measurements_resolver_pipeline_en.md:3]: @akrztrk update here

File: docs/_posts/bunyamin-polat/2024-01-17-cpt_procedures_measurements_resolver_pipeline_en.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+supported: true
+annotator: PipelineModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pipeline employs `sbiobert_base_cased_mli` Sentence Bert Embeddings to map diverse `procedures` and `measurements` entities, to Current Procedural Terminology (CPT) codes.
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-17-cpt_procedures_measurements_resolver_pipeline_en.md:21]: @akrztrk update here

File: docs/_posts/bunyamin-polat/2024-01-17-cpt_procedures_measurements_resolver_pipeline_en.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+val result = ner_pipeline.annotate("""She was admitted to the hospital with chest pain and found to have bilateral pleural effusion, the right greater than the left. CT scan of the chest also revealed a large mediastinal lymph node.
+We reviewed the pathology obtained from the pericardectomy in March 2006, which was diagnostic of mesothelioma.
+At this time, chest tube placement for drainage of the fluid occurred and thoracoscopy, which were performed, which revealed epithelioid malignant mesothelioma.""")
+
+```
+</div>
+
+## Results
+
+```bash
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-17-cpt_procedures_measurements_resolver_pipeline_en.md:61]: @akrztrk shorten the results

File: docs/_posts/bunyamin-polat/2024-01-17-hcc_resolver_pipeline_en.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
+---
+layout: model
+title: Pipeline for HCC Sentence Entity Resolver
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-17-hcc_resolver_pipeline_en.md:3]: @akrztrk update here

File: docs/_posts/bunyamin-polat/2024-01-17-hcc_resolver_pipeline_en.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+supported: true
+annotator: PipelineModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This advanced pipeline extracts `PROBLEM` entities from clinical texts and utilizes the `sbiobert_base_cased_mli` Sentence Bert Embeddings to map these entities to their corresponding Hierarchical Condition Categories (HCC) codes.
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-17-hcc_resolver_pipeline_en.md:21]: @akrztrk update here

File: docs/_posts/bunyamin-polat/2024-01-17-snomed_findings_resolver_pipeline_en.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
+---
+layout: model
+title: Pipeline for Snomed Concept, CT version
```
Comment (muhammetsnts) [inline docs/_posts/bunyamin-polat/2024-01-17-snomed_findings_resolver_pipeline_en.md:3]: @bunyamin-polat ~Pipeline for Snomed Concept, CT version`~ : Pipeline for Snomed Concept, Findings Version

**Review by muhammetsnts — CHANGES_REQUESTED** [review]
@akrztrk @bunyamin-polat @SKocer please check here

File: docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_oncology_en.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ahmet-mesut) [reply docs/_posts/bunyamin-polat/2024-01-16-explain_clinical_doc_oncology_en.md:19]: updated

**Pull request merged**